CONTRIBUTING.md

```diff
@@ -39,12 +41,11 @@ git checkout -b `your-branch-name`
 ### Make commits to your feature branch. 
 
 Prefix each commit like so
-  - (feat) Added a new feature
-  - (fix) Fixed inconsistent tests [Fixes #0]
+  - (feat) Add a new feature
```

Comment:
kinda long but :+1: 
remember, we're trying to show them the the way that is most conducive to collaboration